0232-Implement-Queue-using-Stacks.py

```diff
@@ -45,7 +45,7 @@ def pop(self) -> int:
         Removes the element from in front of queue and returns that element.
         """
         self.switchStack()
-        return None if not self.myStack2 else self.myStack2.pop()
+        return None if not self.myStack2 else self.myStack2.pop() #Does it really needed to be checked if the myStack2 is empty or not?
```

Comment:
According to question NO because in question it is mentioned that peek and pop method will always be called if there are values in the stack. But in general, it should be checked.

Reply:
Ok, thank you!.